go.mod

```diff
@@ -32,6 +32,7 @@ require (
 	github.com/cosmos/ibc-go/v8 v8.4.0
 	github.com/cosmos/rosetta v0.0.0-00010101000000-000000000000
 	github.com/ethereum/go-ethereum v1.14.13
+	github.com/golang/mock v1.6.0
```

Comment:
Should we not be using the gomock fork maintained by Uber?

>Update, June 2023: This repo and tool are no longer maintained. Please see [go.uber.org/mock](https://github.com/uber/mock) for a maintained fork instead.

Reply:
I had to add this to control SDK's mocks :/

Reply:
Great 🎉